lib/iex/lib/iex/autocomplete.ex

```diff
@@ -440,15 +445,17 @@ defmodule IEx.Autocomplete do
   end
 
   defp get_module_types(mod) do
-    cond do
-      not ensure_loaded?(mod) ->
+    with true <- ensure_loaded?(mod),
```

Comment:
Can we please convert this to two `case`s instead of a `with`? The reason is because we are matching on different results `false` and `:error`, I would rather keep it close to the caller instead of flattening everything in a `with`.

Reply:
Since `ensure_loaded` returns a boolean, i replaced it with an `if` and a `case`. Is that ok?